templates/parachain/runtime/src/configs/mod.rs

```diff
@@ -162,7 +162,7 @@ impl pallet_balances::Config for Runtime {
 
 parameter_types! {
 	/// Relay Chain `TransactionByteFee` / 10
-	pub const TransactionByteFee: Balance = 10 * MICROUNIT;
```

Comment:
I can revert the style change if reviewers don't like it.
I change these just because my editor warns me this is a typo